locales/en-US/app.ftl

```diff
@@ -1,8 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 ### Localization for the App UI of Profiler
 
-# Naming convention for l10n IDs: "module-ComponentName--string-summary".
+# Naming convention for l10n IDs: "ComponentName--string-summary".
 # This allows us to minimize the risk of conflicting IDs throughout the app.
-# Please sort alphabetically by (module name, component name). For each module,
+# Please sort alphabetically by (component name).
 # keep strings in order of appearance.
 
-intro = Web app for Firefox performance analysis
+## The following feature names must be treated as a brand. They cannot be translated
+-brand-name = Firefox
+-profiler-brand-name = Firefox Profiler
+
+# Home page translations
```

Comment:
This should be a section commen, since it applies to all the following strings.

Reply:
I have replaced `# Home page translations` with `## Translation Strings`. As we will add all the ftl strings in this file for now I thought it will be better not to dividie these by pages rather add these in alphabetical order of the components